src/HomeWorkApp_1.java

```diff
@@ -0,0 +1,57 @@
+public class HomeWorkApp_1 {
+    public static void main(String[] args) {
+//      printThreeWords();
+//      checkSumSign();
+//      printColor();
+//      compareNumbers();
+
+    }
+
+    public static void printThreeWords(){
+        System.out.println("Orange");
+        System.out.println("Banana");
+        System.out.println("Apple");
+    }
+
+    public static void checkSumSign(){
+        int a;
+        int b;
+        int sum;
+
+        a = 5;
+        b = -8;
+        sum = a + b;
```

Comment:
нет смысла разделять объявление переменной и её инициализацию значением
```suggestion
        int a = 5;
        int b = -8;
        int sum = a + b;
```